docs/docs/midi.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Rotary encoder support
+parent: Porting guide
+nav_order: 6
+redirect_from:
+    - /gettingStarted/midi.html
+    - /getting-started/midi.html
```

Comment:
the redirect isn't needed here, it's only for pages which previously were linked under gettingStarted

Reply:
DONE!